images/base/Dockerfile

```diff
@@ -19,15 +19,15 @@
 
 # start from ubuntu 19.10, this image is reasonably small as a starting point
 # for a kubernetes node image, it doesn't contain much we don't need
-FROM ubuntu:19.10
+FROM ubuntu:20.04
 
 # Configure containerd and runc binaries from kind-ci/containerd-nightlies repository
 # The repository contains latest stable releases and nightlies built for multiple architectures
 ARG CONTAINERD_VERSION="v1.3.3-14-g449e9269"
 # Configure CNI binaries from upstream
 ARG CNI_VERSION="v0.8.5"
```

Comment:
we'll come back for this when it's out

Reply:
no probl, I  was expecting a release this week, but it wasn't the case obviously :)